package.json

```diff
@@ -185,6 +184,10 @@
     "webpack-dev-server": "^3.11.3",
     "webpack-merge": "^5.1.1"
   },
+  "engines": {
+    "node": "16.15.1",
+    "npm": "8.11.0"
+  },
```

Comment:
This produces error when the node and npm version does not match exactly. Developer has to use the exact version. Maybe, we are fine with `16.x` and `8.x`.

Reply:
I think the error is intentional and we want to use the exact version to avoid any harder to debug errors. It should be easy to install and change the version of node if you use nvm.